src/app/auth/store/effects.ts

```diff
@@ -1,77 +1,77 @@
+import { Router } from "@angular/router";
+import { Actions, createEffect, ofType } from "@ngrx/effects";
+import { catchError, map, of, switchMap, tap } from "rxjs";
+import { PersistenceService } from "../../shared/services/persistence.service";
+import { AdminUserInterface } from "../../shared/types/adminUser.interface";
+import { AuthService } from "../services/auth.service";
+import { authActions } from "./actions";
 
 export const getCurrentUserEffect = createEffect(
   (
     actions$ = inject(Actions),
     authService = inject(AuthService),
-    persistanceService = inject(PersistanceService)
+    persistenceService = inject(PersistenceService)
   ) => {
     return actions$.pipe(
       ofType(authActions.getCurrentUser),
       switchMap(() => {
-        const token = persistanceService.get('accessToken')
+        const token = persistenceService.get("accessToken");
 
         if (!token) {
-          return of(authActions.getCurrentUserFailure())
+          return of(authActions.getCurrentUserFailure());
         }
         return authService.getCurrentUser().pipe(
           map((currentUser: AdminUserInterface) => {
-            return authActions.getCurrentUserSuccess({currentUser})
+            return authActions.getCurrentUserSuccess({ currentUser });
           }),
           catchError(() => {
-            return of(authActions.getCurrentUserFailure())
+            return of(authActions.getCurrentUserFailure());
           })
-        )
+        );
       })
-    )
+    );
   },
-  {functional: true}
-)
+  { functional: true }
+);
 
 export const ssoEffect = createEffect(
   (
     actions$ = inject(Actions),
     authService = inject(AuthService),
-    persistanceService = inject(PersistanceService)
+    persistenceService = inject(PersistenceService)
   ) => {
     return actions$.pipe(
-      ofType(authActions.sso),
-      switchMap(({request}) => {
+      ofType(authActions.sSO),
+      switchMap(({ request }) => {
         return authService.sso(request).pipe(
           map((currentUser: AdminUserInterface) => {
-            persistanceService.set('accessToken', currentUser.token)
-            return authActions.ssoSuccess({currentUser})
+            persistenceService.set("accessToken", currentUser.token);
+            return authActions.sSOSuccess({ currentUser });
           }),
           catchError((errorResponse: HttpErrorResponse) => {
             return of(
-              authActions.ssoFailure({
+              authActions.sSOFailure({
                 errors: errorResponse.error.errors,
               })
-            )
+            );
           })
-        )
+        );
       })
-    )
+    );
   },
-  {functional: true}
-)
+  { functional: true }
+);
 
 export const redirectAfterSSOEffect = createEffect(
   (actions$ = inject(Actions), router = inject(Router)) => {
     return actions$.pipe(
-      ofType(authActions.ssoSuccess),
+      ofType(authActions.sSOSuccess),
```

Comment:
use ssoSuccess and ssoFailure instead of sSOSuccess and sSOFailure

Reply:
done